defaults/main.yml

```diff
@@ -97,7 +97,7 @@ env_virt_install: {}
 # - "--numatune 1"
 
 # Specify an os-variant if you want
-os_variant: "rhel6"
+#os_variant: "rhel6"
```

Comment:
From man virt-install: `--os-variant OS_VARIANT
           Optimize the guest configuration for a specific operating system (ex.  'fedora18', 'rhel7', 'winxp'). While not required, specifying this options
           is HIGHLY RECOMMENDED, as it can greatly increase performance by specifying virtio among other guest tweaks. By default, virt-install will attempt to auto detect this value from the install media (currently only supported for URL installs). Autodetection
           can be disabled with the special value 'none'. Autodetection can be forced with the special value 'auto'.`
I don't know if this is an improvement or not, maybe we should warn people if the variable is not set.